### ISSUE TITLE
Like button on post

### ISSUE THREAD

**Issue by Akryum** [posva/onlysponsors#15]
It would allow readers to send a very simple feedback to the author and also make non-sponsors want to read the posts a bit more.

Example implementation I really like:

[![image](https://user-images.githubusercontent.com/2798204/104091135-b1a88b00-527b-11eb-8402-c692001259b4.png)](https://www.joshwcomeau.com/blog/year-in-review-2020/#q2-the-pandemic-and-an-injury)

**Comment by posva** [comment]
Yeah, this would be nice, def worth adding. What website was that?

**Comment by Akryum** [comment]
Click on the image
